portfolio/src/main/webapp/js/script.js

```diff
@@ -11,19 +11,44 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+var mode = false;
 
 /**
- * Switches Light Mode of Page
+ * Sets Light mode of page based on cookies
  */
-function switchMode() {
-  var style = document.getElementById('pagestyle').getAttribute('href');
-  if (style == '') { 
+function setMode() { 
+  var value = getCookie('mode');
+  if(value=="dark"){
```

Comment:
Multiple style violations. Expected state: `if (value == 'dark') {`

1. Need space after `if`
2. Need spaces around `==`
3. Need space after `)`
4. Use single quotes for strings.